src/System.IO.Pipelines/src/System/IO/Pipelines/PipeOptions.cs

```diff
@@ -50,7 +58,7 @@ public class PipeOptions
             WriterScheduler = writerScheduler ?? PipeScheduler.ThreadPool;
             PauseWriterThreshold = pauseWriterThreshold;
             ResumeWriterThreshold = resumeWriterThreshold;
-            MinimumSegmentSize = minimumSegmentSize;
```

Comment:
Note that there was no validation being done on minimumSegmentSize, and I haven't added any.